### PR TITLE
Generate 3d primitives using Buttons

### PR DESCRIPTION
**Pull request by KH-Kyle**
Users can click on different buttons to generate specific 3D primitives (Cube, Sphere, Cylinder, Quad) at a specific location. 

## Changelog

- Added Dashboard prototype prefab 
- Added 3D primitives prefab
- Added script to generate 3D primitives

## Demo

Screen Shot 2022-01-28 at 23 40 54

## Testing

Tested on Quest 2

## Known Issues

See https://github.com/UWRealityLab/xrcapstone22wi-team8/issues/12

### CODE REVIEW

**Comment by KH-Kyle** [comment]
## TODO
- Refine the UI of Dashboard when merging with the brushes.
- Fix Dashboard relative to the camera.

These would be done with teammates before MVP

File: Boardless/Assets/Scripts/ObjectGenerator.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public class ObjectGenerator : MonoBehaviour
+{
+    // The object to generate.
+    public GameObject obj;
+
+    // The position of the generated object.
+    public Vector3 pos; 
+
+    // Generates the preset game object at the given position.
+    public void generate()
```
Comment (ApolloZhu) [inline Boardless/Assets/Scripts/ObjectGenerator.cs:14]: public members should capitalize first letter per coding convention: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#pascal-case

Comment (ApolloZhu) [inline Boardless/Assets/Scripts/ObjectGenerator.cs:14]: public members should capitalize the first letter per coding convention: https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#pascal-case

**Review by ApolloZhu — CHANGES_REQUESTED** [review]
The prefabs look exactly as named, and the dashboard looks like the included demo 👍 

**However**, you should update variable and function names to match the coding standards. Remember to double-check to make sure nothing is broken after renaming 😉

**Review by ApolloZhu — APPROVED** [review]

**Pull request merged**